### DISCUSSION TITLE
Update black carbon to be hydrophobic (currently assumed hydrophytic in CARMA)

### DISCUSSION THREAD

**Discussion by tilmes** [NCAR/CARMA_dev#4] [trop_strat CARMA model]
Description: all CARMA model versions / compsets have assumed that black carbon is hydrophilic, but after conversations with Xiaohong, it should be hydrophobic. MAM also assumes that BC is hydrophobic. Changes have a very small impact on the model results.

change in code tag:  /glade/work/fvitt/camdev/carma_trop_strat  

Needed changes in namelists for all compsets using CARMA:
bin_defs: all places where black-c is set, like this:
black-c:core:/glade/p/cesmdata/cseg/inputdata/atm/cam/physprops/bcphi_rrtmg_carma_c080918.nc:+',
needs to be changed to:
...  black-c:core:/glade/p/cesmdata/cseg/inputdata/atm/cam/physprops/bcpho_rrtmg_carma_c100508.nc:+',

Needed changes in the code: 
/glade/work/fvitt/camdev/carma_trop_strat05/src/chemistry/carma_aero/aero_model.F90
<img width="1139" alt="image" src="https://user-images.githubusercontent.com/22222754/200935070-9e6e098e-17ca-4993-af0f-f3634bc19ea2.png">

**Comment by fvitt** [comment]
Set solmmr to 0.0 for black-c

**Comment by fvitt** [comment]
@tilmes Since you are multiplying by `0.0` doesn't make sense to just remove the line has I have here?

```
diff --git a/src/chemistry/carma_aero/aero_model.F90 b/src/chemistry/carma_aero/aero_model.F90
index 31feb019..627cf1b2 100644
--- a/src/chemistry/carma_aero/aero_model.F90
+++ b/src/chemistry/carma_aero/aero_model.F90
@@ -890,7 +890,6 @@ contains
                          solmmr =  solmmr + raer(mm)%fld(i,k)*0.5_r8
                       end if
                       if (trim(spectype) == 'black-c') then
-                         solmmr =  solmmr + raer(mm)%fld(i,k)*0.1_r8
                          specmmr = specmmr + raer(mm)%fld(i,k)
                       end if
                       if (trim(spectype) == 'p-organic') then
```

**Reply by tilmes** [reply]
This would be fine with me!

**Comment by cbardeen** [comment]
In CARMA, we only have BC in the mixed group. I don’t think that Pengfei agrees with the idea that you find significant amounts of hydrophobic BC in the atmosphere. For CARMA I think by defintion the BC is already hydrophilic when it is introduced since it is part of the mixed group. I think you should check with Pengfei and Yunqian and check his papers before changing this assumption. For MAM it is different as you have the primary carbon that can be hydrophobic. I think if you want hydrophobic BC in CARMA you will need to introduce another group.

> On Nov 16, 2022, at 5:30 PM, Francis Vitt ***@***.***> wrote:
> 
> 
> @tilmes <https://github.com/tilmes> Since you are multiplying by 0.0 doesn't make sense to just remove the line has I have here?
> 
> diff --git a/src/chemistry/carma_aero/aero_model.F90 b/src/chemistry/carma_aero/aero_model.F90
> index 31feb019..627cf1b2 100644
> --- a/src/chemistry/carma_aero/aero_model.F90
> +++ b/src/chemistry/carma_aero/aero_model.F90
> @@ -890,7 +890,6 @@ contains
>                           solmmr =  solmmr + raer(mm)%fld(i,k)*0.5_r8
>                        end if
>                        if (trim(spectype) == 'black-c') then
> -                         solmmr =  solmmr + raer(mm)%fld(i,k)*0.1_r8
>                           specmmr = specmmr + raer(mm)%fld(i,k)
>                        end if
>                        if (trim(spectype) == 'p-organic') then
> —
> Reply to this email directly, view it on GitHub <https://github.com/NCAR/CARMA_dev/discussions/4#discussioncomment-4162438>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AFLRNREGVVAVGCJNFSWCRFTWIV4AXANCNFSM6AAAAAAR32VFQI>.
> You are receiving this because you are subscribed to this thread.
> 

--------------------------------------------------------------
Charles Bardeen
National Center for Atmospheric Research
P.O. Box 3000, Boulder, CO 80307-3000
Phone: (303) 497-1752,  Fax: (303) 497-1400
***@***.***

**Reply by tilmes** [reply]
@cbardeen I checked with Xiaohong earlier, he was insistent that BC should be always hydrophobic, and if we assume it is hydrophilic in CARMA then that is wrong. While Pengfei did this differently, I don't think we should copy something that is wrong. 
Regarding BC in CARMA, while BC is part of the mixed group and therefore hydrophilic, I think, in the wet removal routine (the above code is for that), and also for the radiation code, I would still argue we should treat it hydrophobic. MAM4 also assumes mixed aerosol particles, and for both the primary mode and the accumulation mode BC is treated as hydrophobic in the code. My suggestion therefore is to do the same in CARMA. Do you agree?

**Reply by tilmes** [reply]
@bardeen can @fvitt move forward with this?

**Comment by cbardeen** [comment]
What is kappa for BC in Pengfei’s code?

> On Nov 18, 2022, at 9:28 AM, tilmes ***@***.***> wrote:
> 
> 
> @bardeen <https://github.com/bardeen> can @fvitt <https://github.com/fvitt> move forward with this?
> 
> —
> Reply to this email directly, view it on GitHub <https://github.com/NCAR/CARMA_dev/discussions/4#discussioncomment-4178712>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AFLRNRE4ZSILISLJMTHDLZTWI6VBVANCNFSM6AAAAAAR32VFQI>.
> You are receiving this because you were mentioned.
> 

--------------------------------------------------------------
Charles Bardeen
National Center for Atmospheric Research
P.O. Box 3000, Boulder, CO 80307-3000
Phone: (303) 497-1752,  Fax: (303) 497-1400
***@***.***

**Reply by tilmes** [reply]
here he assumes 0.1, so pretty small:

<img width="1067" alt="image" src="https://user-images.githubusercontent.com/22222754/202755864-444d1195-9b30-4865-8c48-db44ffb5d6e9.png">

@fvitt  This is what is in the current sandbox ( Francis already removed BC here)
<img width="626" alt="image" src="https://user-images.githubusercontent.com/22222754/202756293-c5bea317-cba5-4768-be9d-7e3812829639.png">

**Comment by cbardeen** [comment]
So I would stay consitent with that, but I haven’t looked at where it is used in the hertfrz code. Seems a little strange to me to say BC is hydrophobic, but can also be an IN. Saying it is weakly hydrophilic and a bad IN seems more consitent with what we have assumed in the past. 

However, I would talk to Pengfei and Brian before making a code change.

> On Nov 18, 2022, at 9:38 AM, tilmes ***@***.***> wrote:
> 
> 
> here he assumes 0.1, so pretty small:
> 
>  <https://user-images.githubusercontent.com/22222754/202755864-444d1195-9b30-4865-8c48-db44ffb5d6e9.png>
> —
> Reply to this email directly, view it on GitHub <https://github.com/NCAR/CARMA_dev/discussions/4#discussioncomment-4178794>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AFLRNRG5C7FJJ3Y4EAGJOX3WI6WIHANCNFSM6AAAAAAR32VFQI>.
> You are receiving this because you were mentioned.
> 

--------------------------------------------------------------
Charles Bardeen
National Center for Atmospheric Research
P.O. Box 3000, Boulder, CO 80307-3000
Phone: (303) 497-1752,  Fax: (303) 497-1400
***@***.***

**Comment by fvitt** [comment]
These changes are implemented in tag carma_trop_strat09
